metis-enrichment/metis-enrichment-client/src/test/java/eu/europeana/enrichment/rest/client/EnrichmentWorkerTest.java

```diff
@@ -23,23 +40,10 @@
 import org.mockito.ArgumentCaptor;
 import org.mockito.Captor;
 import org.mockito.Mockito;
-import org.mockito.runners.MockitoJUnitRunner;
-import eu.europeana.corelib.definitions.jibx.RDF;
-import eu.europeana.enrichment.api.external.model.Agent;
-import eu.europeana.enrichment.api.external.model.EnrichmentBase;
-import eu.europeana.enrichment.api.external.model.EnrichmentResultList;
-import eu.europeana.enrichment.api.external.model.Place;
-import eu.europeana.enrichment.api.external.model.Timespan;
-import eu.europeana.enrichment.rest.client.EnrichmentWorker.Mode;
-import eu.europeana.enrichment.utils.EntityClass;
-import eu.europeana.enrichment.utils.EntityMergeEngine;
-import eu.europeana.enrichment.utils.InputValue;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.web.client.RestTemplate;
 
-// TODO Tests that could be added:
-// 1. What if there is nothing to process (empty or null lists are returned when extracting or when
-// enriching/dereferencing).
-// 2. Throwing exceptions
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
Usage of the silent mode is NOT recommended. Why not use the default MockitoJUnitRunner behaviour?

Reply:
Fixed.